README.md

```diff
@@ -30,6 +30,10 @@ $ ...repeat with each *.test executable...
 - `libps.a` is the output of the library compilation.  Including this in your `-L` path, and including this directory in `-I` will enable you to use the library.
 - `README.md` and `TODO.md` ...take a wild guess ;-)
 - `plat/` is the platform directory including both architecture-specific, and OS-specific functions.
+- `quiesce_type/` is a directory including different quiescence calculation implementations (See RTAS paper for detail).
+  - `general/` is optimized for best effort enviroment;
+  - `real_time/` provides guarantee for response time and memory utilization, and can be used in soft/hard real-time system;
+  - `temporal/` can only be used in hard real-time system, as it requires to konw maximal response time in parallel section apriori.
```

Comment:
spelling.  I've said this before, I'll implore you again:  please add spellcheck in emacs (flyspell mode).